Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+
+For the specified devices, we enable XDP acceleration for maximum efficiency.
+The vast majority of modern network drivers for 10/40/100G and beyond support
+XDP today. If you are unsure whether your network driver supports native XDP,
+then simply set ``--bpf-lb-acceleration=disabled`` instead and Cilium loads
+its programs into the tc BPF layer instead of XDP.
+
+.. parsed-literal::
+
+   docker run --name cilium-gateway -td \\
+     -v /sys/fs/bpf:/sys/fs/bpf \\
+     -v /lib/modules:/lib/modules \\
+     --privileged=true \\
+     --network=host \\
+     quay.io/cilium/cilium:|IMAGE_TAG| \\
+     cilium-agent \\
+     --enable-ipv4=true \\
+     --enable-ipv6=true \\
+     --datapath-mode=lb-only \\
+     --bpf-lb-algorithm=maglev \\
+     --bpf-lb-acceleration=native \\
+     --devices=bond0
+
+By default, the layer 4 load-balancer operates in Source NAT (SNAT) mode, meaning replies from
+backends will be returned to the load-balancer which will then perform reverse NAT and
+sends the reply back to the client. Advanced options such as Direct Server Return
+(DSR) are supported as well and described in later sections of this guide.
+
+Validate the Setup
+==================
+
+After deploying Cilium Standalone Gateway with the Quick-Start guide above, we can first
+validate that the Cilium agent is running in the desired mode:
+
+.. parsed-literal::
+
+   docker exec cilium-gateway cilium status --verbose
+   [...]
+   Kubernetes:             Disabled
+   KubeProxyReplacement Details:
+     Status:                 Partial
+     Socket LB:              Enabled
+     Socket LB Protocols:    TCP, UDP
+     Devices:                bond0 86.109.5.207 2604:1380:4091:cf00::1 (Direct Routing)
+     Mode:                   SNAT
+     Backend Selection:      Maglev (Table Size: 16381)
+     Session Affinity:       Disabled
+     Graceful Termination:   Enabled
+     NAT46/64 Support:
+     - Services:             Enabled
+     - Gateway:              Disabled
+     XDP Acceleration:       Native
+   [...]
+
+As an optional next step, we will create a simple service entry for testing purpose.
+Then we will validate that Cilium installed the service correctly.
+
+Cilium exposes its programmable API under ``unix:///var/run/cilium/cilium.sock``.
+Cilium's `service command-line interface <https://github.com/cilium/cilium/blob/master/cilium/cmd/service_update.go>`_
+communicates to the agent through this API. For simplicity, we use the
+command-line tool in this example, but third party orchestration tooling can use
+the API by connecting to the unix domain socket directly. For the latter, it is
+recommended to expose Cilium's runtime directory to the host via Docker volumes
+(``-v /var/run/cilium/:/var/run/cilium/``), so that other control plane software
+can talk to ``/var/run/cilium/cilium.sock``.
+
+After initial deployment the load-balancer service table is empty:
+
+.. parsed-literal::
+
+  docker exec cilium-gateway cilium service list
```

Comment:
Incidentally I'm not sure that the indentation is quite right here? Typically we suggest 4-space indentation for all of these sphinx snippets. I think that for the first layer of nesting it might not matter, but if you ever need to double-nest sphinx output then you may need to follow the 4-space indentation consistently to get the hunks to format right.